test_driver.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I think just remove the `cd` commands and move this script into the `test_integration` folder. I would like to keep the root as clean as possible.

Reply:
Done